Core/include/Acts/Propagator/EigenStepper.ipp

```diff
@@ -84,12 +84,12 @@ void Acts::EigenStepper<E, A>::update(State& state,
 
 template <typename E, typename A>
 void Acts::EigenStepper<E, A>::update(State& state, const Vector3& uposition,
-                                      const Vector3& udirection, double up,
+                                      const Vector3& udirection, double qop,
```

Comment:
I'm guessing you checked all uses of this function to make sure we're passing in the right value?

Reply:
I checked again and it seems to be only used in `PointwiseMaterialInteraction` and `SimulationActor`

Reply:
Don't we use it in the CKF too?

Reply:
only through `PointwiseMaterialInteraction`. the `CKF` is using the other overloads

Reply:
ok